main.js

```diff
@@ -1598,50 +1647,47 @@ async function createWindow() {
 
     async function windowDiscordSettings() {
         const discord = new BrowserWindow({
-            //parent: mainWindow,
+            parent: settings,
             icon: iconDefault,
             modal: false,
             frame: windowConfig.frame,
-            titleBarStyle: windowConfig.titleBarStyle,
+            titleBarStyle: 'default',
```

Comment:
But we're forcing user choice already to the 'nice' style which is deficient in what it displays - doesn't display the app title.  What does it give you over the default system styling (which you can change from)?

Reply:
> Doesn't display the app title

This is indeed a minor thing to be missing, but given that this was something from the start of the project and how the application has been show.
To my knowledge there was works to get this to likely be themeable in CSS so that it could match your theme / album art. (This was present in the insperation fo this from GPMDP)

Reply:
So why show the 'Nice' theme as a default if it:
a) Doesn't show the app title at the top (like the system one does)
b) Is not the Electron default title bar which should ideally be more compatible/same across platforms

Reply:
I'm not backing down from this.
When you show off a product you show it off generally with all the bells and whistles to make it visually look good.
The user obviously can go through the settings (And generally considered all users will as there's so many other additions which YTM Desktop offers in features)
That's when they can decide if they want to go back to the System title bar or not.
Changes like this are the sort of things which are discussed by the community and not an individual, before being actioned.

Ultimately this is going to be the last opportunity to get a v1 release before we finalise up the final bits we want for v2 and ship that and v1 is no longer supported at all (given how much of a mess code base is, building not working, how out of date most dependencies are). V2 has been a complete re-code for modern Electron with far better security measures put in place for it to work with modern practices.

Ultimately there's some bigger changes coming in with how the title bar is managed in v2 using some of Electron's new features for "overlaying" over existing controls to provide a Customisable Native Taskbar (Good example inside of VS Code)
Example on MacOS we may not even have a top bar at all and the "traffic lights" will be embedded into the YT top bar.